llvm/lib/Target/AMDGPU/AMDGPUAttributor.cpp

```diff
@@ -439,6 +439,26 @@ struct AAAMDAttributesFunction : public AAAMDAttributes {
       indicatePessimisticFixpoint();
       return;
     }
+
+    SmallPtrSet<const Constant *, 8> VisitedConsts;
+
+    for (Instruction &I : instructions(F)) {
```

Comment:
Should use checkForAllInstructions instead of manually looking at all instructions